tests/rest/admin/test_user.py

```diff
@@ -1119,7 +1119,7 @@ def test_user_does_not_exist(self):
         self.assertEqual(404, channel.code, msg=channel.json_body)
         self.assertEqual(Codes.NOT_FOUND, channel.json_body["errcode"])
 
-    def test_user_is_not_local(self):
+    def test_user_does_not_exist_and_is_not_local(self):
```

Comment:
The docstring for this test needs to be updated.

We probably want to add new tests for the success path for a remote user.

Reply:
How can I add remote users in a test?

Reply:
It can be done by adding an event to the room that the user joined, something like:

https://github.com/matrix-org/synapse/blob/c619253db80c8d1c606dc40756dd3c9e3a55a9fb/tests/test_visibility.py#L150-L171